qdrant-landing/content/documentation/concepts/explore.md

```diff
@@ -16,22 +16,36 @@ In addition to the regular search, Qdrant also allows you to search based on mul
 REST API - API Schema definition is available [here](https://api.qdrant.tech/api-reference/search/recommend-points)
 
 ```http
-POST /collections/{collection_name}/points/recommend
+POST /collections/{collection_name}/points/query
 {
-  "positive": [100, 231],
-  "negative": [718, [0.2, 0.3, 0.4, 0.5]],
-  "filter": {
-        "must": [
-            {
-                "key": "city",
-                "match": {
-                    "value": "London"
-                }
-            }
+  "query": {
+    "recommend": {
+      "positive": [
+        100,
+        231
+      ],
+      "negative": [
+        718,
+        [
+          0.2,
+          0.3,
+          0.4,
+          0.5
         ]
+      ],
```

Comment:
In cases like this I'd prefer to join lines to prevent making the snippets very long (just like we had before):

```suggestion
      "positive": [100, 231],
      "negative": [718, [0.2, 0.3, 0.4, 0.5]],
```